apps/crn-server/test/integration/common/reminders.integration-test.ts

```diff
@@ -0,0 +1,583 @@
+import supertest from 'supertest';
+import { Express } from 'express';
+import {
+  ListReminderResponse,
+  UserCreateDataObject,
+  UserResponse,
+} from '@asap-hub/model';
+
+import { appFactory } from '../../../src/app';
+import { retryable } from '../../helpers/retryable';
+import {
+  FixtureFactory,
+  getUserFixture,
+  UserFixture,
+  getEventFixture,
+  getTeamFixture,
+  TeamFixture,
+  EventCreateDataObject,
+  EventFixture,
+} from '../fixtures';
+import { getCalendarFixture } from '../fixtures/calendar';
+
+jest.mock('../../../src/config', () => ({
+  ...jest.requireActual('../../../src/config'),
+  isContentfulEnabledV2:
+    process.env.INTEGRATION_TEST_CMS === 'contentful' ? 'true' : undefined,
+  logLevel: 'silent',
+}));
+
+jest.setTimeout(120000);
+
+const fixtures = FixtureFactory(process.env.INTEGRATION_TEST_CMS);
+
+type AppResponse<T> = {
+  status: number;
+  body: T;
+};
+
+describe('Reminders', () => {
+  let app: Express;
+  let loggedInUser: UserFixture;
+  let notLoggedInUser: UserFixture;
+  let team: TeamFixture;
+
+  beforeAll(async () => {
+    await fixtures.clearAllPreviousEvents();
+    jest.useFakeTimers({ doNotFake: ['setTimeout'] });
+    team = await fixtures.createTeam(getTeamFixture());
+    loggedInUser = await fixtures.createUser(
+      getUserFixture({
+        firstName: 'LoggedIn',
+        email: 'loggedIn@user.com',
+        teams: [
+          {
+            id: team.id,
+            role: 'Co-PI (Core Leadership)',
+          },
+        ],
+      }),
+    );
+    notLoggedInUser = await fixtures.createUser(
+      getUserFixture({
+        firstName: 'NotLoggedIn',
+        email: 'notloggedIn@user.com',
+        teams: [
+          {
+            id: team.id,
+            role: 'Collaborating PI',
+          },
+        ],
+      }),
+    );
+    app = appFactory({
+      authHandler: (req, _res, next) => {
+        req.loggedInUser = loggedInUser as UserResponse;
+        next();
+      },
+    });
+  });
+
+  afterAll(async () => {
+    jest.useRealTimers();
```

Comment:
don't we want to teardown everything, ie use ```await fixtures.teardown();```

Reply:
committed here [add teardown](https://github.com/yldio/asap-hub/pull/3236/commits/04bd546215dda035793a31f6a812b6b09314ade0)